README.md

```diff
@@ -189,28 +192,28 @@ All passwords used for your computing devices, Rapid River resources (such as em
 * be changed at least annually and not reuse any previous passwords
 * may not contain the user’s account name or any part of their full name
 
-We strongly suggest using a password locker such as [LastPass](https://lastpass.com) or [1Password](https://1password.com/) to simplify the management and rotation of passwords in order to comply with the requirements listed above.
+We strongly suggest using a password locker such as [LastPass](https://lastpass.com) or [1Password](https://1password.com/) to simplify the management and rotation of passwords to comply with the requirements listed above.
 
 ### Network security
 
-Whether you work from home or an office, we need the network on which you work to be secure. For that reason:
+Whether you work from home or in an office, we need the network on which you work to be secure. For that reason:
 
-* WiFi shall be configured with a minimum of WPA2-PSK or WPA-2 Personal encryption standard
-* All default vendor passwords must be changed prior to connecting any devices for use at Rapid River
-* Router software shall be kept up to date and patched with vendor’s software
-* Routers shall have external access (remote administration) disabled
-* Default SSIDs must be changed from vendor default to something unique
-* We prefer you to configure guest WiFi features on your router, if it supports it
+* WiFi shall be configured with a minimum of WPA2-PSK or WPA-2 Personal encryption standard.
```

Comment:
This is all lifted from the IB security policies, so you're correcting their lawyers' grammar :-)

Nevertheless, I don't have any problems with this set of changes.